Gemfile.lock

```diff
@@ -1,15 +1,15 @@
 PATH
   remote: .
   specs:
-    pg_query (2.1.3)
+    pg_query (2.1.4)
       google-protobuf (>= 3.19.2)
 
 GEM
   remote: https://rubygems.org/
   specs:
     ast (2.4.1)
     diff-lcs (1.3)
-    google-protobuf (3.20.0)
+    google-protobuf (3.21.2)
```

Comment:
I did this in a separate PR (and to a slightly newer version) for clarity: https://github.com/pganalyze/pg_query/pull/253/files